requests-for-comments/2022-11-08-supported-languages.md

```diff
@@ -0,0 +1,169 @@
+# Request for comments: Supported languages at PostHog
+
+Since day 1 here at Posthog we've supported two and half core languages: Python and Javascript/Typescript.
+
+This set of tools has carried us a long way and we should both the cpython runtime and node a huge hats off for bringing us this far.
+
+## Problem statement
+
+We have the opportunity coming up to rebuild or greenfield build out services that are critical parts of our data pipeline. It will be important for these services to be correct, fast, and efficient. Considering this now is a good time to ask: Are we using the correct tools for the job?
+
+So why even bring this up?
+
+Frankly:
+
+- Python is slow.
+- Node is a memory hog.
```

Comment:
Can you please expand on this and mention at least 2 use cases from our codebase where Node memory management was the bottleneck? (note: **Node memory management**, not how we write Node).